Exercise_1.java

```diff
@@ -1,44 +1,65 @@
-class Stack { 
-    static final int MAX = 1000; 
-    int top; 
-    int a[] = new int[MAX]; // Maximum size of Stack 
-  
-    boolean isEmpty() 
-    { 
-        //Write your code here 
-    } 
-
-    Stack() 
-    { 
-        //Initialize your constructor 
-    } 
-  
-    boolean push(int x) 
-    { 
-        //Check for stack Overflow
-        //Write your code here
-    } 
-  
-    int pop() 
-    { 
-        //If empty return 0 and print " Stack Underflow"
-        //Write your code here
-    } 
-  
-    int peek() 
-    { 
-        //Write your code here
-    } 
-} 
-  
-// Driver code 
-class Main { 
-    public static void main(String args[]) 
-    { 
-        Stack s = new Stack(); 
-        s.push(10); 
-        s.push(20); 
-        s.push(30); 
-        System.out.println(s.pop() + " Popped from stack"); 
-    } 
-}
+class Exercise_1 { 
+	    static final int MAX = 1000; 
+	    int top; 
+	    int a[] = new int[MAX];
+	  
+	    boolean isEmpty() 
+	    {
+	    	if (top<0){
+	    		return true;
+	    	}
+			return false; 
+	    } 
+
+	    Exercise_1() 
+	    { 
+	        top=-1;
+	    } 
+	  
+	    boolean push(int x) 
+	    { 
+	        if (top>=MAX){
```

Comment:
This condition will give an exception because the last index for the array is MAX-1, So if the top is equal to MAX-1, we will get an exception